src/d3a/global_utils.py

```diff
@@ -0,0 +1,10 @@
+from d3a.constants import BidOfferMatchAlgoEnum
```

Comment:
Why does this need to be a separate module?

Reply:
This is to solve some imports issue, it used to reside in a directory 2 levels below this one, and we had to import it from different directories in the same level

Reply:
I am not a fan of adding yet another util module. And I really dont understand the import issues. 
Can you try to add it to `d3a/d3a-core/util.py` please?